potpie/cli/commands/start.py

```
@@ -0,0 +1,91 @@
+        print(f"Error: {exc}", file=sys.stderr)
+        sys.exit(1)
+    start_script = project_root / "scripts" / "start.sh"
+
+    if start_script.exists():
+        if not start_script.is_file():
+            print(f"Error: {start_script} is not a regular file.", file=sys.stderr)
+            sys.exit(1)
+        print("Starting Potpie server…")
+        try:
+            subprocess.run(  # noqa: S603 # NOSONAR — trusted script from project tree, no shell, fixed args
+                ["bash", str(start_script)],
```

Ruff (0.15.6)

[error] Starting a process with a partial executable path

(S607)